crates/wasm-smith/src/config.rs

```diff
@@ -129,6 +129,12 @@ pub trait Config: 'static + std::fmt::Debug {
         100
     }
 
+    /// Export all WebAssembly objects in the module. This overrides
+    /// [`Config::min_exports`] and [`Config::max_exports`]. Defaults to false.
+    fn export_everything(&self) -> bool {
+        false
+    }
```

Comment:
Could this be added to `SwarmConfig` as well? (probably `false` during the `Arbitrary` impl though)